README.md

```diff
@@ -9,16 +9,19 @@ Brownfield project for Turing students to collaborate on as a full-stack team in
 
 Original **Crate Repo** [README](https://github.com/turingschool/Crate)
 
-### Team Members
+### About
+
+Crate is a platform, similar to Stitch Fix, that allows users to setup monthly subscriptions to receive trendy clothes and accessories.
+
+### Technologies Used
+
 **Front End**
-* [Erin Untermeyer](https://github.com/ErinUntermeyer)
-* [Nicole Latifi](https://github.com/NicoleLatifi)
-* [Corbin March](https://github.com/MarchCorbin)
+- React
+- Redux
 
 **Back End**
-* [Kwibe Merci](https://github.com/jKwibe)
-* [Gaby Mendez](https://github.com/gabichuelas)
-* [Ryan Laleh](https://github.com/RyN21)
+- GraphQL
+- Sequelize
```

Comment:
We should probably add Express and Node.js here, but I can do it when I update the GraphQL schema image.